Frontend/Arc-i-Tech/src/app/login/login.component.css

```diff
@@ -0,0 +1,136 @@
+.box {
+    width: 90%;
+    max-width: 600px;
+    min-height: 500px;
+    margin: 2% auto;
+    border-radius: 10px;
+    background-color: #112D4E !important;
+    box-shadow: rgba(0, 0, 0, 0.25) 0px 54px 55px, rgba(0, 0, 0, 0.12) 0px -12px 30px, rgba(0, 0, 0, 0.12) 0px 4px 6px, rgba(0, 0, 0, 0.17) 0px 12px 13px, rgba(0, 0, 0, 0.09) 0px -3px 5px;
+    padding: 20px;
+}
+h2 {
+    text-align: center;
+    margin-bottom: 20px;
+    margin-top: 20px;
+    font-size: 2.5em;
+    color: #F9F7F7 !important;
+}s
+.form-fields {
+    margin-bottom: 15px;
+}
+.form-fields label {
+    display: block;
+    color: #F9F7F7 !important;
+    margin-bottom: 5px;
+    margin-top: 30px;
+    font-size: 1.2em;
+    padding-left: 10%;
+}
+.form-fields input[type="text"],
+input[type="password"] {
+    width: 80%;
+    padding: 15px;
+    margin-top: 5px;
+    border: 1px solid #ccc;
+    border-radius: 5px;
+    font-size: 16px;
+    margin-left: 10%;
+}
+.form-fields button {
+    width: 40%;
+    height: 50px;
+    margin-top: 50px;
+    margin-bottom: 30px;
+    margin-left: 30%;
+    border-radius: 10px;
+    background-color: #3F72AF;
+    color: white;
+    font-size: 1.2em;
+    border: none;
+    cursor: pointer;
+    transition: background-color 0.3s ease;
+}
+.form-fields button:hover {
+    background-color: #9BB8EE;
+}
+.extra-links {
+    text-align: center;
+    margin-top: 10px;
+    margin-bottom: 10px;
+}
+.extra-links a {
+    color: #007bff;
+    text-decoration: none;
+    padding: 20px;
+    margin: 0 5px;
+}
+.extra-links a:hover {
+    text-decoration: underline;
+}
+/* Small devices */
+@media (max-width: 576px) {
+    .box {
+        width: 100%;
+        padding: 15px;
+    }
+    h2 {
```

Comment:
create the root path  **.box .form-fileds h2{    CSS properties  }**

Reply:
Root path is added